packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js

```diff
@@ -119,51 +120,49 @@ describe('validate record', () => {
         Number_Affordable_Housing_Units__c: 0,
     };
 
-    it('validates a valid project succesfully', () => {
-        return validateRecord({
-            upload: { ec_code: '2.16' },
-            record: VALID_EC2_PROJECT,
-            typeRules: ALL_RULES['ec2'],
-        }).then(
-            (generatedErrors) => {
-                assert(generatedErrors.length == 0,
-                    `Unexpected error when validating record: ${generatedErrors}`)
-            },
-            (thrownException) => { fail('Unexpected error while validating record', thrownException) }
-        );
-    });
+    it('validates a valid project succesfully', () => validateRecord({
+        upload: { ec_code: '2.16' },
+        record: VALID_EC2_PROJECT,
+        typeRules: ALL_RULES.ec2,
+    }).then(
+        (generatedErrors) => {
+            assert(generatedErrors.length === 0,
+                `Unexpected error when validating record: ${generatedErrors}`);
+        },
+        (thrownException) => { fail('Unexpected error while validating record', thrownException); },
```

Comment:
@trumanc Any chance you remember where this `fail` function should be coming from?

I noticed that `chai` and `assert` libraries both provide a `fail` method, but neither signature seems to match your usage here.

Reply:
oh whoops, it looks like I meant for that to be a single string that included the exception info, not a second parameter that breaks the method signature. You could change this to

```(thrownException) => { fail(`Unexpected error while validating record: ${thrownException}`); },```

Reply:
Awesome, thanks for clarifying!